### PR TITLE
Custom class fixes

### PR DESCRIPTION
**Pull request by numidium**
This fixes the "reputation change" dialog which was previously not appearing. I also changed the StatsRollout on the class creator window to behave like it does in classic.

### CODE REVIEW

**Comment by ajrb** [comment]
Oops looks like we have both made the same changes to StatsRollout tonight. @Interkarma I will mark the parts of this PR that are already in my PR so you can decide what to do about this when merging. Sorry for the inconvenience

File: Assets/Scripts/Game/UserInterfaceWindows/CreateCharCustomClass.cs [hunk 1]
```diff
@@ -401,7 +401,7 @@ void ExitButton_OnMouseClick(BaseScreenComponent sender, Vector2 position)
             }
 
             // Are all attribute points distributed?
-            if (statsRollout.BonusPool > 0) 
+            if (statsRollout.BonusPool != 0)
```
Comment (ajrb) [inline Assets/Scripts/Game/UserInterfaceWindows/CreateCharCustomClass.cs]: Duplicated in my PR #965

File: Assets/Scripts/Game/UserInterface/StatsRollout.cs [hunk 2]
```diff
@@ -234,8 +234,8 @@ void Spinner_OnUpButtonClicked()
             // Get working stat value
             int workingValue = workingStats.GetPermanentStatValue(selectedStat);
 
-            // Working value cannot rise above maxWorkingValue and bonus cannot fall below zero
-            if (workingValue == maxWorkingValue || bonusPool == 0)
+            // Working value cannot rise above maxWorkingValue and bonus cannot fall below zero if not in free edit mode
+            if (workingValue == maxWorkingValue || (bonusPool == 0 && !freeEdit))
```
Comment (ajrb) [inline Assets/Scripts/Game/UserInterface/StatsRollout.cs]: Duplicated in my PR #965

**Comment by numidium** [comment]
Resolved the conflict and went with @ajrb's code since it was more complete.

**Comment by ajrb** [comment]
Thanks for doing that, sorry about the conflict I was just working through bug reports.

**Pull request merged**